src/backend/distributed/planner/deparse_shard_query.c

```diff
@@ -486,14 +499,25 @@ DeparseTaskQuery(Task *task, Query *query)
 
 
 /*
- * TaskQueryString generates task->queryStringLazy if missing.
+ * TaskQueryStringAllPlacements generates task->queryStringLazy if missing.
  *
  * For performance reasons, the queryString is generated lazily. For example
  * for local queries it is usually not needed to generate it, so this way we
  * can skip the expensive deparsing+parsing.
  */
 char *
-TaskQueryString(Task *task)
+TaskQueryStringAllPlacements(Task *task)
```

Comment:
Can't we just call TaskQueryStringForAllPlacements directly?

Reply:
Yeah, I initially had something different here so now it makes sense to call TaskQueryStringForAllPlacements directly.